plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -219,12 +219,15 @@ public Version getVersion() throws IOException {
      * @throws InterruptedException
      *         if build process was interrupted
      */
-    public String buildImage(String repository, ProgressMonitor progressMonitor, AuthConfigs authConfigs, File... files)
-            throws IOException, InterruptedException {
+    public String buildImage(String repository,
+                             ProgressMonitor progressMonitor,
+                             AuthConfigs authConfigs,
+                             boolean doForcePull,
```

Comment:
Yes i agree with @benoitf, it would be really nice to use objects in the case of `DockerConnector` methods.
But, as it requires a lot of changes you can add another method with a new argument and reuse this method in old one, which will save API compatibility.